packages/bento-design-system/src/CheckboxGroupField/createCheckboxGroupField.tsx

```diff
@@ -74,8 +74,13 @@ export function createCheckboxGroupField(
             assistiveTextProps={descriptionProps}
             errorMessageProps={errorMessageProps}
           >
-            <Inset spaceY={config.group.paddingY}>
-              {(props.orientation || "vertical") === "vertical" ? (
+            <Inset
+              spaceY={
+                ((config.group.internalSpacing[props.orientation ?? "vertical"] as number) /
+                  2) as ComponentProps<typeof Inset>["spaceY"]
+              }
```

Comment:
@veej this fails at runtime if the `spacing/2` isn't a valid spacing, but allowing a custom spacing for `Inset` would make its interface less strict and I think it isn't something we want.

Reply:
I think you can easily replace the `Inset` with a raw `Box` passing a `style` to bypass the design tokens